docs/en/01.overview.md

```diff
@@ -0,0 +1,192 @@
+
+
+
+Damocles focuses on implementing a set of efficient sealing cluster solutions that accomplish goals including
+
+- When faced with many homogeneous computing devices, reduce the number of types of hardware required, the number of instances, and the complexity of deployment.
+- Safely complete "self-healing" for abnormalities in various stages and of different types as much as possible, reducing manual intervention and improving production efficiency.
+- Utilize cloud services, outsourcing services, and other sharable infrastructure to lower thresholds of entry into the industry and reduce unnecessary costs.
+
+
+
+Of course, this clear and targeted design may make this set of solutions not universally suitable for all users. For example, if the user already owns a large number of devices configured separately for different stages of sealing forming appropriate combinations, then the Damocles solution may not effectively improve the sealing efficiency. The user can judge the specific circumstances wherein using Damocles is appropriate after reading subsequent documentations.
+
+
+
+### Lotus sealing component mechanism
+
+Before introducing Damocles-specific design approaches, we need to review the mechanism of Lotus-related components.
+
+The Lotus sealing components form a standard centralized scheduling cluster:
+
+- `lotus-miner` is the central scheduling service that allocates and manages tasks.
+- `lotus-seal-worker` is the sealing execution body, responsible for completing the specific tasks assigned to it and providing feedback to `lotus-miner`.
+
+`lotus-miner`, as the initiator, interacts with `lotus-seal-worker` via RPC.
+
+
+
+The advantages of this design are obvious:
+
+- `lotus-seal-worker` is an almost stateless working process that is theoretically easy to scale out horizontally.
+- `lotus-miner` can flexibly schedule tasks among `lotus-worker` instances.
+
+
+
+But this mechanism also has another side:
+
+1. `lotus-miner`, as the scheduling core:
+
+   - Bears heavy and different types of logic, such as task scheduling and management, interacting with the chain, etc.
+   - Retains the historical burden of having to be compatible with both local mode and remote mode at the same time.
+  
+   These lead to complex internal logic (state machine) and difficulty in refining exception handling.
+
+2. `lotus-seal-worker` is designed to be completely stateless.
+
+
+
+This has led to some problems, such as:
+
+1. Due to the vastly different hardware requirements between stages, in order to ensure reasonable resource allocation, `lotus-seal-worker` is usually configured so that each instance only supports tasks from one stage, and multiple different instances are deployed on the same machine, isolated through OS-level resource control (cgroups, docker, numa, etc.).  
```

Comment:
```suggestion
1. Due to the vastly different hardware requirements between stages, in order to ensure reasonable resource allocation, `lotus-worker` is usually configured so that each instance only supports tasks from one stage, and multiple different instances are deployed on the same machine, isolated through OS-level resource control (cgroups, docker, numa, etc.).  
```